documentation/Windows Defender scan DetectionHistory file format.asciidoc

```diff
@@ -45,6 +45,7 @@ in the section entitled "GNU Free Documentation License".
 |===
 | Version | Author | Date | Comments
 | 0.0.1 | J.B. Metz | February 2022 | Initial version.
+| 0.0.2 | Fry | July 2023 | Additional information.
```

Comment:
@Fryyyyy are you sure you want this to be Fry ?

Reply:
I'll also change 2023 to 2022, I know time flies but not that fast

Reply:
Thanks :) 
Yes, Fry is preferred